frontend/src/metabase/admin/datamodel/metadata/components/MetadataTableColumn/MetadataTableColumn.tsx

```diff
@@ -52,7 +60,11 @@ const MetadataTableColumn = ({
 
   const handleChangeDescription = useCallback(
     (event: ChangeEvent<HTMLInputElement>) => {
-      onUpdateField({ id: field.id, description: event.target.value });
+      if (event.target.value) {
```

Comment:
q: when it's possible to have event.target.value falsy? does it cover empty value aka `''`?

Reply:
empty string